decidim-core/app/packs/src/decidim/redesigned_external_link.js

```diff
@@ -17,12 +17,18 @@ export default class ExternalLink {
       externalLink: "External link"
     };
 
-    this.setup(node);
+    /* REDESIGN_PENDING: this is done to avoid duplication due to the polling,
```

Comment:
Yes, otherwise the behaviour is applied multiple times. This is because the initializer is applied over the whole document in the first load, then on the list of comments once the comments have been loaded, and then again each time the polling ends.